build.gradle

```diff
@@ -5,15 +5,19 @@ buildscript {
     ext.jna_version = '5.2.0'
     ext.android_gradle_plugin_version = '3.6.0'
 
-    // N.B. try to keep these two in sync
-    ext.android_components_version = '24.0.0'
-    ext.glean_version = '21.3.0'
+    // Since the Glean version depends on the Android components version,
+    // it is very important to use a modern version of Glean and, ideally,
+    // let this come from the embedding product itself.
```

Comment:
micro-nit: it's not clear to me if a future reader of this file will know how to interpret this comment, since there's now nothing in the surrounding code about glean. I wonder if it would be more likely to serve its intended purpose if moved down to the part where we use the glean gradle plugin.

Reply:
Good point. I moved this down, where I declare the glean plugin dependency!